code/datums/spells/cluwne.dm

```diff
@@ -5,7 +5,7 @@
 
 	school = "transmutation"
 
-	charge_max = 600
+	base_cooldown = 1 MINUTES
 	clothes_req = TRUE
 	cooldown_min = 200 //100 deciseconds reduction per rank
```

Comment:
```suggestion
	cooldown_min = 20 SECONDS //100 deciseconds reduction per rank
```

Reply:
feel like this is out of scope. I dont wanna change these really, I started but I gave up quickly. for a future PR